life_expectancy/tests/test_cleaning.py

```diff
@@ -1,15 +1,16 @@
 """Tests for the cleaning module"""
 import pandas as pd
 
-from life_expectancy.cleaning import DATA_DIR, FILE_NAME, FILE_PATH, main
+from life_expectancy.main import FILE_NAME, main
 
-from . import OUTPUT_DIR
+from . import FIXTURES_DIR, OUTPUT_DIR
 
 
 def test_clean_data(pt_life_expectancy_expected):
```

Comment:
Shouldn't this be `test_main` (i.e, an integration test)? You don't need to save the data when you test the cleaning function.